client/src/components/Search.js

```diff
@@ -30,18 +33,29 @@ class Search extends React.Component {
                     asFormData: false
                 });
                 if (form) {
-                    this.props.fetchUser(form, methods.SEARCH_USERS);
+                    this.props.fetchSearch(methods.SEARCH_USERS, form);
                 } else {
                     console.log('form invalid')
                 }
             }
+            if(tagName === 'I') {
+                if(dataset.friend == 'true') {
+                    console.log(dataset.friend)
```

Comment:
remove all console.log from the commit. It's not a good practice to push it to the PR